### ISSUE TITLE
Experiment page Layout - discussion

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#139]
When I open the experiment page, the Custom View button is overlaid to the cord slice. To avoid that, I need to make the 3D viewer taller by stretching it upward, but this makes the 2D viewer smaller (too small) and clips the controllers un the left part of the 3D viewer. 
Possible solutions:
1) in the 3D viewer, move the slide more to the right (so not at the center) such that the Custom View button does not cover it;
2) try to but the 2D viewer and the 3D viewers side-by-side as opposed to over each other but this might complicate things because the cord occupies space horizontally and not vertically. 

thoughts?

<img width="834" alt="Screenshot 2022-06-15 at 10 50 59" src="https://user-images.githubusercontent.com/57769656/173857854-2f5ab80a-9b3f-48a4-91a8-056dba8805ca.png">

**Comment by afonsobspinto** [comment]
I don't have that behavior by default: 
![image](https://user-images.githubusercontent.com/19196034/173861011-fac309a8-4c9c-496a-af72-a4fc084e9389.png)

I'm guessing this is something that might be dependent on the resolution of the screen. Maybe with some better style rules we can make it good for everyone. What do you think @vidhya-metacell ?

**Comment by zsinnema** [comment]
@afonsobspinto yes it depends on the size of the flex layout tabs and they depend on the size of the screen.... 

@neuroBazzi I already mentioned this before to you and also Paul... I suggest to NOT overlap the elements and give the elements "room" to dispay. I mean f.e. in the 2D viewer give split the tab laout container into 2 parts: one for the popup and one for the images. now it's one container where the popup is positioned on top of the images

**Comment by neuroBazzi** [comment]
@zsinnema understood and agree
